packages/ppx/src/declarations_to_emotion.re

```diff
@@ -270,7 +276,10 @@ let transform_with_variable = (parser, mapper, value_to_expr) =>
     fun
     | `Variable(name) => render_variable(name)
     | `Value(ast) => mapper(ast),
-    value_to_expr,
+    fun
+    | {pexp_desc: Pexp_ident({txt: Ldot(Lident("CssJs"), _), _}), _} as exp => value_to_expr(exp) // we dont want to add types on cases with `CssJs.red` for example
```

Comment:
I don't fully understand the comment, or why we are branching for idents.

Reply:
`transform_with_variables` is called from `apply(Parser.spec, [% CssJs.fn], map_fn)` and it generates a function call `CssJs.fn(map_fn(parsed_spec))`.

The `map_fn` usually returns polymorphic variants, so when it is an `Pexp_ident` its because it is using a local value or a value from some module, this would be interpolation, except when it is some value directly from `CssJs`, for example `CssJs.red, CssJs.blue`, since we dont want to add type notation into those, we have the branch where if the module is `CssJs` we just call `value_to_expr` without adding types.

Reply:
Right, so you only want to render the type when `Variable is there?

Reply:
Yes, since `Variable calls `render_variable` that returns a `Pexp_ident`.

Matching `Variable` directly in `emit` didn't worked because the typechecker would complain that `Variable` polyvar didn't exists in some types generated by spec parser.

Doing it in the mapping function would not work too because it would type the arg instead of the function